app/src/main/java/com/simply/birthdayapp/auth/signIn/domain/model/LoginInput.kt

```diff
@@ -0,0 +1,7 @@
+package com.simply.birthdayapp.auth.signIn.domain.model
+
+data class LoginInput(
```

Comment:
you are using same model for domain/data layers please separate and create 2 models for each